shepherd_bms/include/datastructs.h

```diff
@@ -20,7 +20,7 @@ struct ChipData_t
     //These are calculated during the analysis of data
     uint8_t cellTemp[NUM_CELLS_PER_CHIP];
```

Comment:
```suggestion
    int8_t cellTemp[NUM_CELLS_PER_CHIP];
```
Oops this probably should be signed